.travis.yml

```diff
@@ -12,6 +12,7 @@ python:
   - "2.7"
 before_script:
   - sleep 10
+  - ./.setup_travis
```

Comment:
rather than having another file that clutters the project, just have an echo of your `.vent-startup.yml` here instead:

```
echo $'https://github.com/cyberreboot/vent:\n  rabbitmq:\n    build: yes\n    start: yes' > ~/.vent-startup.yml'
```